tasks/build.py

```diff
@@ -135,6 +135,38 @@ def create_directory(pr, jobs_base_dir, event_info):
     return ym, pr_id, run_dir
 
 
+def run_cmd(run_command, cmd_detail, arch_job_dir):
```

Comment:
Some suggestions:

```python
def run_cmd(cmd, log_msg='', parent_dir=None):

    if parent_dir is None:
        parent_dir = os.getcwd()

    if log_msg:
      log("'%s' by running '%s' ..." % (log_msg, cmd, ...))
    else:
      log("Running '%s' ..." % (cmd, ...))
```

Reply:
I wonder if we really need the `log_msg`. Should be enough to simply log the command that is run.